src/Microsoft.AspNet.Mvc.Razor/IRazorPage.cs

```diff
@@ -67,5 +68,7 @@ public interface IRazorPage
         /// <exception cref="InvalidOperationException">if one or more sections were not rendered or if no sections were
         /// defined and the body was not rendered.</exception>
         void EnsureRenderedBodyOrSections();
+
+        IHtmlContentBuilder Output { get; set; }
```

Comment:
What's the difference here between `BodyContent` and `Output`.

In the earlier code, I know that `RenderBodyDelegateAsync()` would render the body. What triggers rendering of the body now?